prow/oss/plugins.yaml

```diff
@@ -338,33 +341,11 @@ plugins:
     - project # Lets issues be tagged into projects
     - size
     - skip  # Allows cleaning up stale commit statuses
+    - transfer-issue # Transfer issue to a different repo in the same org.
+    - trigger # Enable /ok-to-test command to trigger GitHub actions with /rerun-all
```

Comment:
@andreyvelich if my understanding is correct, `/ok-to-test` should trigger GitHub Actions after [this change](https://github.com/GoogleCloudPlatform/oss-test-infra/pull/2331/files#diff-61d4ecf8ed653ce1a03b8a3758c9fa61af90500fd05f6580dbafcfef3ec780f0R345). But it didn't.
![Screenshot from 2024-08-13 12-57-51](https://github.com/user-attachments/assets/3ad5ed89-351b-4422-8317-0dbc1dd62483)

https://github.com/kubeflow/pipelines/pull/11096#issuecomment-2286585861

Do you know if there's anything else that we should do, or is that a bug?

cc @rimolive @DharmitD

Reply:
@hbelmiro This plugin just adds `ok-to-test` label to the PR after we add this comment: `/ok-to-test`.
It doesn't trigger to GitHub actions.
Since we use this action to trigger our action: https://github.com/estroz/rerun-actions, you need to add this comment to trigger the actions:
```
/rerun-all
```

Reply:
I see.
The comment made me think that `/ok-to-test` would somehow automatically trigger `/rerun-all`.
Thank you @andreyvelich.